packages/mui-material/src/Popper/Popper.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import PopperUnstyled, { PopperUnstyledProps } from '@mui/core/PopperUnstyled';
+import { chainPropTypes, HTMLElementType, refType } from '@mui/utils';
+import { Direction, useThemeWithoutDefault as useTheme } from '@mui/system';
+
+export type PopperProps = PopperUnstyledProps;
+
+/**
+ *
+ * Demos:
+ *
+ * - [Autocomplete](https://mui.com/components/autocomplete/)
+ * - [Menus](https://mui.com/components/menus/)
+ * - [Popper](https://mui.com/components/popper/)
+ *
+ * API:
+ *
+ * - [Popper API](https://mui.com/api/popper/)
+ * - inherits [PopperUnstyled API](https://mui.com/api/popper-unstyled/)
+ */
+const Popper = React.forwardRef(function Popper(
+  props: PopperProps,
+  ref: React.ForwardedRef<HTMLDivElement>,
+) {
+  const theme = useTheme<{ direction?: Direction }>();
+  const themeDirection = {
+    direction: theme?.direction,
+  };
+
+  return <PopperUnstyled {...themeDirection} {...props} ref={ref} />;
```

Comment:
```suggestion
  return <PopperUnstyled direction={theme.direction} {...props} ref={ref} />;
```

Theme should never be undefined/null even if there is nothing in the context.

Reply:
useTheme does actually return undefined in regression tests, so we'll have to do this instead:
```jsx
<PopperUnstyled direction={theme?.direction} {...props} ref={ref} />
```

Reply:
It is a bug then, `useTheme` should never return undefined if used from the `@mui/material/styles` package.

Reply:
Ah, I see it is imported from the system. Ok nevermind